src/wiktextract/extractor/el/pos.py

```diff
@@ -599,7 +598,7 @@ def extract_form_of_templates(
     # * Only handle cases where the second argument refers to a form:
     #   μορφ / μορφή / λόγια μορφή του, etc.
     #   and ignore those mistakenly used as synonym templates
-    if t_name == "γρ" and 2 in t_node.template_parameters:
+    if t_name in ("γρ", "γραφή του") and 2 in t_node.template_parameters:
```

Comment:
[Πρότυπο:alter](https://el.wiktionary.org/w/index.php?title=%CE%A0%CF%81%CF%8C%CF%84%CF%85%CF%80%CE%BF:alter&redirect=no) also redirect to [Πρότυπο:γρ](https://el.wiktionary.org/wiki/%CE%A0%CF%81%CF%8C%CF%84%CF%85%CF%80%CE%BF:%CE%B3%CF%81). Template redirects can be checked at [here](https://el.wiktionary.org/wiki/%CE%95%CE%B9%CE%B4%CE%B9%CE%BA%CF%8C:%CE%A4%CE%B9%CE%A3%CF%85%CE%BD%CE%B4%CE%AD%CE%B5%CE%B9%CE%95%CE%B4%CF%8E?target=%CE%A0%CF%81%CF%8C%CF%84%CF%85%CF%80%CE%BF%3A%CE%B3%CF%81&namespace=10&hidetrans=1&hidelinks=1&limit=50).